streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/TaskAssignorTest.java

```diff
@@ -46,7 +46,7 @@ public void testAssignWithoutStandby() {
 
         // # of clients and # of tasks are equal.
         tasks = mkSet(0, 1, 2, 3, 4, 5);
-        assignments = TaskAssignor.assign(states, tasks, 0);
+        assignments = TaskAssignor.assign(states, tasks, 0, null);
```

Comment:
@guozhangwang 

I thought the SLF4J formatting handled null values, but to be sure I tried running the test with `INFO` turned on - resulted in:  `INFO stream-thread [null]  ....`  no NPE but not very helpful, maybe I should still set a value for more meaningful information ? Also do the same for the other tests with dummy null values ?